homework-submission/mobx-week1/src/stores/TodoStore.js

```diff
@@ -0,0 +1,40 @@
+import { observable, action, computed } from "mobx";
```

Comment:
You missed This 👌
```suggestion
import { observable, action, computed, configure } from "mobx";
configure({ enforceActions: 'observed' });
```

You don't have to put it there but for a development purposes it's preferred to be here always while doing it.